src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,162 @@
+---
+title: APM Glossary
+sidebar_order: 5
+---
+
+## Clock Skew
+
+If you are collecting transactions from more than a single machine, you will likely encounter **clock skew**. When collecting and comparing transaction events that are gathered from different devices, you may observe timestamps that do not align or events in a trace that don't share a time history. Sentry does not attempt to correct the timestamps in your events. The timestamps displayed in Sentry for transactions and each span retain the original values reported by your application/hosting environment.
+
+While you can reduce clock skew by utilizing Network Time Protocol (NTP) or your cloud provider's clock synchronization services, you may still notice small drifts in your data, as synchronizing clocks on small intervals is challenging.
+
+## Performance Metrics
+
+As part of APM and transaction data, we’re exposing a few new aggregate metrics to help customers better understand the performance of their applications.
+
+### Duration Percentiles
+
+The average, 75%, 95%, and 99% duration, help guide customers in identifying transactions that are slower than the user’s target SLAs.
+
+Our APM data is sampled, so the percentiles we present only represent the data we have. Because of sampling, data ranges, filters, or a low traffic transaction, we will often have a case where data is directionally correct, but not accurate. For example, the average of one number is that number, but that’s not usually what people expect to see.
+
+This inability to be usefully accurate will happen more often in some columns than others. We can calculate an average with less data than a 95th percentile, but it’ll also vary by row. An index page will always get more traffic than `/settings/country/belgium/tax`.
+
+### Percentage Query
+
+This column shows the percentage of the entire query that this row represents, so you filter your results with the query and then see how much of the result set a particular transaction represents.
+
+### Requests Per Minute (RPM)
+
+Requests Per Minute is a way to measure throughput. It is the average of all requests bucketed by the minute for the current time window and query string.
+
+## Span
+
+The **span** is the primary building block of a distributed trace, representing an individual unit of work done in a distributed system. It can be the processing of an HTTP request by a server, the querying of a database, or an HTTP call to an external service.
```

Comment:
```suggestion
The **span** is the primary building block of a distributed trace, representing an individual unit of work done in a distributed system. It may represent the processing of an HTTP request by a server, the querying of a database, or an HTTP call to an external service.
```